src/schema/artist/carousel.ts

```diff
@@ -31,7 +32,7 @@ const ArtistCarousel = {
         top_tier: true,
       }),
       artistArtworksLoader(id, {
-        size: 7,
+        size: 10, // we only show a max of 7 though, a hotfix for AS-285
```

Comment:
👍 I guess there's a small risk that the overfetch will _still_ not get enough eligible artworks, but if that's the case it would bespeak a more serious problem with the data.